readme.md

```diff
@@ -18,11 +18,28 @@ var stripCssComments = require('strip-css-comments');
 stripCssComments('/*! <copyright> */ body { /* unicorns */color: hotpink; }');
 //=> '/*! <copyright> */ body { color: hotpink; }'
 
-// use the `all: true` option to strip everything
-stripCssComments('/*! <copyright> */ body { /* unicorns */color: hotpink; }', {all: true});
-//=> ' body { color: hotpink; }'
-```
+//assign the preserve option `false` to strip all comments including `/*!`
+stripCssComments(
+	'/*! <copyright> */ body { /* unicorns */color: hotpink; }', 
+	{preserve: false}
+);
+//=> 'body { color: hotpink; }'
+
+//assign the preserve option a regular expression to strip comments not matching the pattern
+stripCssComments(
+	'/*! <copyright> */ body { /* unicorns */color: hotpink; }', 
+	{preserve: /^\\!/}
```

Comment:
Sorry about this. My Markdown preview uses JavaScript and was requiring me to add an extra escape character. I meant to remove them.